src/Symfony/Component/Routing/Tests/Generator/UrlGeneratorTest.php

```diff
@@ -662,6 +662,14 @@ public function testFragmentsDoNotEscapeValidCharacters()
         $this->assertEquals('/app.php/testing#?/', $url);
     }
 
+    public function testFragmentsCanBeDefinedAsDefaults()
+    {
+        $routes = $this->getRoutes('test', new Route('/testing', array('_fragment' => 'fragment')));
+        $url = $this->getGenerator($routes)->generate('test', array(), true);
```

Comment:
The third argument must not be a boolean. Please pass `UrlGeneratorInterface::ABSOLUTE_PATH` here (see #20009 which fixes the other tests).